### ISSUE TITLE
Attendance report with power BI

### ISSUE THREAD

**Issue by Akki11234** [Akki11234/questions#1]
I am trying to build a Power BI report with respect to attendance. I am new to power BI and i am finding it hard to arrive at the correct logic.

So my data looks something like this

<img width="170" alt="image" src="https://github.com/Akki11234/questions/assets/133892630/aeb15b0d-8c0d-4e80-8bc1-f6ca9c709b58">

And I need an output like this

<img width="252" alt="image" src="https://github.com/Akki11234/questions/assets/133892630/7799c461-47cd-4212-8107-94d571622b8c">

So basically, The days attended columns should contain the number of days attended based on the date mentioned. For example, No 3 attended on 10th of May first. So his value in days attended column is 1 and he attended again on 13th, so his value becomes 2

Can someone please help. I've been stuck with this for quite a while now
